zerver/openapi/zulip.yaml

```diff
@@ -11746,6 +11746,194 @@ paths:
       responses:
         "200":
           $ref: "#/components/responses/SimpleSuccess"
+  /invites:
+    get:
+      operationId: get-invites
+      summary: Get all invites
+      tags: ["invites"]
+      description: |
+        Fetch all the invitations (i.e. email invitations and reusable invitation
+        links) that can be controlled by the user.
+
+        This isn't necessarily the same as all the invitations generated by the
+        user, as administrators can also control invitations that they did not
+        themselves create.
+
+        **Changes**: Prior to Zulip 8.0 (feature level 209), non-admin users could
+        only create email invitations, and therefore the response did not include
+        reusable invitation links for these users.
+      responses:
+        "200":
+          description: Success.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/JsonSuccessBase"
+                  - additionalProperties: false
+                    properties:
+                      result: {}
+                      msg: {}
+                      ignored_parameters_unsupported: {}
+                      invites:
+                        type: array
+                        description: |
+                          An array of `invites` object.
```

Comment:
This should be:

> An array of objects, each representing a single unexpired invitation.

(I checked the implementation, and expired invitations are not returned).